[2024-11-28T20:10:02.997+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-28T20:10:03.024+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Forecast.fetch_forecast_weather manual__2024-11-28T20:10:01.413610+00:00 [queued]>
[2024-11-28T20:10:03.031+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Forecast.fetch_forecast_weather manual__2024-11-28T20:10:01.413610+00:00 [queued]>
[2024-11-28T20:10:03.031+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-28T20:10:03.043+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): fetch_forecast_weather> on 2024-11-28 20:10:01.413610+00:00
[2024-11-28T20:10:03.058+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Weather_Forecast', 'fetch_forecast_weather', 'manual__2024-11-28T20:10:01.413610+00:00', '--job-id', '233', '--raw', '--subdir', 'DAGS_FOLDER/weather_forecast_dag.py', '--cfg-path', '/tmp/tmp_fs99shz']
[2024-11-28T20:10:03.060+0000] {standard_task_runner.py:91} INFO - Job 233: Subtask fetch_forecast_weather
[2024-11-28T20:10:03.061+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=349) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-28T20:10:03.061+0000] {standard_task_runner.py:63} INFO - Started process 352 to run task
[2024-11-28T20:10:03.102+0000] {task_command.py:426} INFO - Running <TaskInstance: Weather_Forecast.fetch_forecast_weather manual__2024-11-28T20:10:01.413610+00:00 [running]> on host fd43cd87157f
[2024-11-28T20:10:03.184+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Weather_Forecast' AIRFLOW_CTX_TASK_ID='fetch_forecast_weather' AIRFLOW_CTX_EXECUTION_DATE='2024-11-28T20:10:01.413610+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-28T20:10:01.413610+00:00'
[2024-11-28T20:10:03.185+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-28T20:10:03.783+0000] {python.py:237} INFO - Done. Returned value was:          Date  temperature_2m_max  ...  windspeed_10m_max  avg_temperature
0  2024-11-28                12.2  ...               23.7             6.50
1  2024-11-29                14.7  ...               17.6             8.85
2  2024-11-30                14.7  ...               14.6            10.50
3  2024-12-01                14.2  ...               25.0             9.40
4  2024-12-02                 6.5  ...               20.2             2.80
5  2024-12-03                 6.3  ...               12.7             2.30
6  2024-12-04                 8.2  ...               15.8             2.80

[7 rows x 7 columns]
[2024-11-28T20:10:03.797+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-28T20:10:03.844+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Weather_Forecast, task_id=fetch_forecast_weather, run_id=manual__2024-11-28T20:10:01.413610+00:00, execution_date=20241128T201001, start_date=20241128T201003, end_date=20241128T201003
[2024-11-28T20:10:03.879+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-28T20:10:03.899+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-28T20:10:03.903+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
